src/vs/workbench/contrib/markdown/browser/media/markdown.css

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/* Alert color mappings using theme variables */
+blockquote[data-severity="note"] {
+	--vscode-textBlockQuote-border: var(--vscode-editorInfo-foreground);
+}
+
+blockquote[data-severity="tip"] {
+	--vscode-textBlockQuote-border: var(--vscode-charts-green);
```

Comment:
A few of these hardcode specific colors while others use more general colors item, such as `--vscode-editorWarning-foreground`. Could all of them use these more broad color names? Maybe we need to introduce some new colors?